CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.1.0)
 
 project(sway C)
 
-set(CMAKE_C_FLAGS "-g")
+set(CMAKE_C_FLAGS "-g -lm")
```

Comment:
I think `target_link_libraries` is the preferred way to link stuff. And since this is only required by sway this could be moved to `sway/CMakeLists.txt`.